democracylab/emails.py

```diff
@@ -27,7 +27,7 @@ class EmailSection(Enum):
 class Html:
     @staticmethod
     def a(href, text):
-        return "<a href={href}>{text}</a>".format(href=href, text=text)
+        return """<a href="{href}">{text}</a>""".format(href=href, text=text)
```

Comment:
Wanted to call this out - I saw several methods to get our href output to include quotes, and I went with this since it was fast and human legible. If another method looks cleaner now's as good a time as any to try it since testing will have to go beyond just the volunteer email since Href.a is used elsewhere.

Reply:
Ordinarily we use single quotes for strings that will include double quotes, and vice versa.  I went ahead and changed this line to that format in my latest change.